addons/markers/functions/fnc_initInsertMarker.sqf

```diff
@@ -213,6 +231,30 @@
     _buttonCancel ctrlSetPosition _pos;
     _buttonCancel ctrlCommit 0;
 
+    ////////////////////
+    // init marker timestamp cb
+
+    if (GVAR(timestampEnabled) && {[ACE_player] call FUNC(canTimestamp)}) then {
+        _buttonOK ctrlAddEventHandler ['ButtonClick', {
+            if (GETUVAR(GVAR(timestampChecked),false)) then {
+                params ["_buttonOk"];
+                private _description = (ctrlParent _buttonOk) displayctrl IDC_INSERT_MARKER;
+
+                _description ctrlSetText format [ // Add timestamp suffix
+                    "%1 [%2]",
+                    ctrlText _description,
+                    [daytime, "HH:MM"] call BIS_fnc_timeToString
```

Comment:
maybe provide a setting for time format and 24/12 h display

Reply:
![image](https://user-images.githubusercontent.com/14627848/95312990-e299ae00-088f-11eb-9fd4-0144c2684fde.png)